frontend/src/app/map-view/map/layers/area-layer.ts

```diff
@@ -267,6 +273,49 @@ class AreaLayer extends VectorLayer<VectorSource> {
     map.addInteraction(areaHover);
   }
 
+  private addRightClickInteraction(map: OLMap) {
+    let path = '';
+    const
+      optionsMenuElement = document.getElementById('area-options-menu')!,
+      remover = () => {
```

Comment:
Change name to something like removeOptionsMenu?

Reply:
Good idea 👍